packages/@aws-cdk/aws-apigateway/README.md

```diff
@@ -565,7 +565,7 @@ books.addMethod('GET', new apigateway.HttpIntegration('http://amazon.com'), {
 });
 ```
 
-You can find a full working example [here](test/authorizers/integ.request-authorizer.ts).
+You can find a full working example [here](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-apigateway/test/authorizers/integ.token-authorizer.ts).
```

Comment:
wondering if it's better to just include the example as this content is not really going to apply to Python, Java, .NET users but the README is published to corresponding package managers.

will defer to @nija-at though as he's the primary maintainer of this area :)